frontend/src/components/NewShoot/GNewShootInfrastructureDetails.vue

```diff
@@ -8,7 +8,7 @@ SPDX-License-Identifier: Apache-2.0
   <v-container class="px-0 mx-0">
     <v-row>
       <v-col
-        v-if="cloudProfiles.length > 1"
+        v-if="cloudProfiles.length > 1 || !cloudProfileHasSeedNames"
```

Comment:
why do we need this change here?

Reply:
If a cloudprovider has only one profile we want to show the dropdown anyway in case there is no seed as this is the only place where we show the hint about missing seeds (hint). If you have a better idea please let me know :)